pkg/cli/run.go

```diff
@@ -323,6 +323,13 @@ func (s *Run) update(ctx context.Context, c client.Client, imageSource imagesour
 		}
 		updateOpts.Image = image
 		updateOpts.DeployArgs = deployArgs
+	} else if len(imageSource.Args) > 0 {
+		imageSource.Image = app.Status.AppImage.Name
+		_, deployArgs, err := imageSource.GetImageAndDeployArgs(ctx, c)
+		if err != nil {
+			return nil, false, err
+		}
+		updateOpts.DeployArgs = deployArgs
```

Comment:
nit: @g-linville what do you think about this alternative: 

```go
if _, updateOpts.DeployArgs, err = imageSource.GetImageAndDeployArgs(ctx, c); err != nil {
    return nil, false, err
}
```

is it too terse?

Reply:
Nope, I like it :+1: